docs/backend-api/how-to/how-to-create-a-rule.md

```diff
@@ -0,0 +1,200 @@
+---
+title: How to use rules
+description: Rules used to set up conditions according to which the system logs the events and sends notifications to user.
+---
+
+# How to use rules
+
+Rules used to set up conditions according to which the system logs the events and sends notifications to user.
+
+When a server receives a new portion of data from the device, it checks whether the conditions set are true 
+or false for this data. If they are true, the server generates an event in history, logs it and immediately sends SMS or
+ Email, as specified.
+ 
+### Create
+
+Let's create a rule with conditions according to which the platform will generate events and schedule intervals when this
+ rule should work. The user must have access to rule update.
+
+#### parameters
+
+| name | description | type |
+| :------ | :------ | :----- |
+| name | The name of created rule. | string |
+| description | Rule's description. | string |
+| zone_ids | List of zones to bind where the rule will work. Leave it empty if rule should work everywhere. | array of int |

```

Comment:
Parameter `zone_ids` is not allowed for rule `offline` and required for `route` and `inoutzone`.